build.sh

```diff
@@ -236,7 +237,10 @@ if [ $target == "test" ]; then
     file_started=test/runs_started.$configure
     file_completed=test/runs_completed.$configure
 
+    ls test/*/test_*.unit 
     ls test/*/test_*.unit                   > $file_attempted
+    head test/*/test_*.unit
+    grep  "BEGIN" test/*/test_*.unit
```

Comment:
Were these 3 new lines just for debugging? Or are they meant to be permanent additions?

Reply:
fixed